src/runtime-rs/crates/runtimes/virt_container/src/lib.rs

```diff
@@ -125,6 +137,38 @@ fn new_agent(toml_config: &TomlConfig) -> Result<Arc<KataAgent>> {
     }
 }
 
+// agent-specific kernel parameters
+fn agent_kparams(toml_config: &TomlConfig) -> Result<Vec<String>> {
```

Comment:
I think it's better to do the operation of handling config file when loading the toml config. Try to put this inside `kata-types/src/config`

Reply:
Thanks for the review, @Tim-0731-Hzt 

The way you suggest will bring unwanted dependencies to libs, but I will try to do this in another way.
- 1. From toml_config, get a kv of agent kernel parameters (this can be done in libs using `HashMap<String, String>`)
- 2. From the kernel parameters kv, get the agent parameters and update this to hypervisor config (this can be done in load_config)